YellowNotes.Api/Startup.cs

```diff
@@ -17,6 +24,29 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
+            AllowedHosts allowed;
+            using (FileStream fs = File.OpenRead("appsettngs.json"))
+                {
+                allowed = await JsonSerializer.DeserializeAsync<AllowedHosts>(fs);
+                }
```

Comment:
definitely not in that way, please take a look at how your friends did it.

Reply:
refactored it into working one